10/umbraco-deploy/deployment-workflow/import-export.md

```diff
@@ -0,0 +1,440 @@
+- Umbraco content (such as content and media) are [transferred by editors using backoffice operations](./content-transfer.md).
+
+We recommend using these approaches for day-to-day editorial and developer activities.
+
+Import and export is intended more for larger transfer options, project upgrades, or one-off tasks when setting up new environments.
+
+As import and export is a two-step process, it doesn't require inter-environment communication. This allows us to process much larger batches of information without running into hard limits imposed by Cloud hosting platforms.
+
+We are also able provide hooks to allow for migrations of data types and property data when importing. This should allow you to migrate your Umbraco data from one Umbraco major version to a newer one.
+
+## Exporting content and schema
+
+To export content and schema, you can select either a specific item of content, or a whole tree or workspace.
+
+When exporting, you can choose to include associated media files.  Bear in mind that including media for a large site can lead to a big zip file.  So even with this option, you might want to consider methods options for transferring large amounts of media. For example using direct transfer between Cloud storage accounts or File Transfer Protocol (FTP).
+
+If your account has access to the Settings section, you can also choose to include the schema information and related files as well.
+
+![Export dialog](./images/export-dialog.png)
+
+Umbraco Deploy will then serialize all the selected items to individual files, archive them into a zip file and make that available for download.  You can download the file using the _Download_ button.
+
+After download you should also delete the archive file from the server. You can do this immediately via the _Delete_ button available on the dialog.
+
+![Export dialog complete](./images/export-dialog-complete.png)
+
+If you miss doing this, you can also clean up archive files from the Umbraco Deploy dashboard in the _Settings_ section.
+
+![Delete exports](./images/delete-exports.png)
+
+## Importing content and schema
+
+Having previously exported content and schema to a zip file, you can import this into a new environment.
+
+![Import dialog](./images/import-dialog.png)
+
+You can upload the file via the browser.
+
+Similar to when exporting, you can choose to import everything from the archive file, or only content, schema or files.
+
+![Import dialog step 2](./images/import-dialog-2.png)
+
+We validate the file before importing.  Schema items that content depends on must either be in the upload itself, or already exist on the target environment with the same details.  If there are any issues that mean the import cannot proceed, it will be reported.  You may also be given warnings for review. You can choose to ignore these and proceed they aren't relevant for the action you are carrying out.
```

Comment:
```suggestion
We validate the file before importing.  Schema items that content depends on must either be in the upload itself or already exist on the target environment with the same details.  If there are any issues that mean the import cannot proceed, it will be reported.  You may also be given warnings for review. You can choose to ignore these and proceed if they aren't relevant to the action you are carrying out.
```